src/components/ui/ButtonToggle.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components/macro'
+
+const Button = styled.button`
```

Comment:
If you style `ResetButton` (which is in UI) here I think you can avoid some of the CSS properties.